launcher/modplatform/ModIndex.h

```diff
@@ -55,6 +56,35 @@ struct DonationData {
     QString url;
 };
 
+struct IndexedVersionType {
+    enum class Enum { Release = 1, Beta, Alpha, UNKNOWN };
```

Comment:
why not something *like* this?
```suggestion
    enum class VersionType { Unknown = 0, Release, Beta, Alpha };
```

Reply:
Renamed the enum as suggested, but kept the order as is.
When sorting the release should be first(if they share the same date) not unknown.